src/test/scala/io/iohk/ethereum/blockchain/sync/LoadableBloomFilterSpec.scala

```diff
@@ -0,0 +1,55 @@
+package io.iohk.ethereum.blockchain.sync
+
+import com.google.common.hash.{Funnel, Funnels, PrimitiveSink}
+import io.iohk.ethereum.FlatSpecBase
+import io.iohk.ethereum.db.dataSource.RocksDbDataSource.IterationError
+import monix.eval.Task
+import monix.reactive.Observable
+
+class LoadableBloomFilterSpec extends FlatSpecBase {
+  implicit object LongFun extends Funnel[Long] {
+    override def funnel(from: Long, into: PrimitiveSink): Unit = {
+      Funnels.longFunnel().funnel(from, into)
+    }
+  }
+
+  "LoadableBloomFilter" should "load all correct elements " in testCaseM[Task] {
+    for {
+      source <- Task(Observable.fromIterable(Seq(Right(1L), Right(2L), Right(3L))))
+      filter <- Task.now(LoadableBloomFilter[Long](1000, source))
```

Comment:
why not `filter = LoadableBloomFilter(...)` then? (and below too)

Reply:
to be honest constant changes between `=` and `<-` just makes code unreadable for me, but sure i will change it 👍